.github/workflows/build.yml

```diff
@@ -0,0 +1,145 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: Build and Test
+
+on:
+  # Capture this event so that gradle caches are updated when a PR is merged to develop
+  # More information on why: https://github.com/gradle/gradle-build-action#using-the-caches-read-only
+  push:
+    branches:
+      - develop
+    paths:
+      - 'src/**'
+      - '.github/**'
+      - 'ansible/**'
+      - 'Docker/**'
```

Comment:
I think these paths are not correct for the REST adapter, only the `.github` one applies. These need to be related to the paths in this repository, same comment regarding the PR block.